playbooks/prerequisite_checks.yml

```diff
@@ -0,0 +1,139 @@
+---
+- name: prerequisite checks
+  hosts: repository
+  become: true
+  become_method: sudo
```

Comment:
sudo is anyway the default become_method and specifying it would make it impossible to run the prereq playbook on systems using su e.g.
As it is now I can't see any other reason other become_method wouldn't work.

Reply:
good chatch, removed the method in both playbooks